src/clean.cc

```diff
@@ -189,21 +189,21 @@ int Cleaner::CleanTargets(int target_count, char* targets[]) {
   LoadDyndeps();
   for (int i = 0; i < target_count; ++i) {
     string target_name = targets[i];
-    uint64_t slash_bits;
-    string err;
-    if (!CanonicalizePath(&target_name, &slash_bits, &err)) {
-      Error("failed to canonicalize '%s': %s", target_name.c_str(), err.c_str());
+    if (target_name.empty()) {
+      Error("empty path");
```

Comment:
Keep `Error("failed to canonicalize '%s': empty path", target_name.c_str());` so that the error stays the same?

Reply:
I actually preserved it originally, but then decided to update it since no other code paths use the `failed to canonicalize` wording.  I've restored it for now so that this PR remains refactoring-only.